### ISSUE TITLE
I think the button element is missing a URL field

### ISSUE THREAD

**Issue by sudermanjr** [slack-go/slack#1085]
From the block kit, I can generate:

```
		{
			"type": "section",
			"text": {
				"type": "mrkdwn",
				"text": "This is a section block with a button."
			},
			"accessory": {
				"type": "button",
				"text": {
					"type": "plain_text",
					"text": "Click Me",
					"emoji": true
				},
				"value": "click_me_123",
				"url": "https://google.com",
				"action_id": "button-action"
			}
		},
```

That button accessory has a `url`, which I can't seem to add to that button. Maybe I'm missing something?

```
github.com/slack-go/slack v0.10.3
```

**Comment by sudermanjr** [comment]
Actually, I was wrong. The field exists, it's just not available in the New helper function.

**Comment by pthieu** [comment]
@sudermanjr can you show what your final solution came to be?

**Comment by sudermanjr** [comment]
This was a while ago, but I am pretty sure I had to construct the block myself, rather than using the helper function.

**Comment by joshbranham** [comment]
Implemented a fix on #1250
